test/bdd/features/activitypub.feature

```diff
@@ -112,7 +112,6 @@ Feature:
     Then the JSON path "type" of the response equals "CollectionPage"
     And the JSON path "items" of the response does not contain "${domain2IRI}"
 
-""" TODO: Enable this test when CIDs for ipfs and local CAS are identical
```

Comment:
@sandrask Actually, per a follow-up discussion with @troyronda it looks like we need to make the CIDs we produce compatible with IPFS now. v1 CIDs are now the default and they work with IPFS. I re-enabled the tests again and they pass.